kickstarts/partials/post/source_setup.ks.erb

```diff
@@ -16,7 +16,7 @@ pushd $appliance_root
 popd
 
 mkdir -p $app_root
-git clone <%= @manageiq_checkout.remote %> $vmdb_root
+git clone --separate-git-dir /tmp/manageiq.git <%= @manageiq_checkout.remote %> $vmdb_root
```

Comment:
What if we clone with `--depth=1` then change the `.git/config` from:
`fetch = +refs/heads/master:refs/remotes/origin/master`
to:
`fetch = +refs/heads/*:refs/remotes/origin/*`